src/absil/ilread.fs

```diff
@@ -1029,377 +1032,373 @@ let count c =
 #endif
         
 
-let seekReadUInt16Adv ctxt (addr: byref<int>) =  
```

Comment:
@ncave I suppose we don't mind a major `#if` in the interim, though it's unfortunate